public class TrafficLight {

    private final String id;
    private String color;

    public TrafficLight(String id) {
        this.id = id;
        this.color = "RED";
    }

    public void next() {
        if (color.equals("RED")) {
            color = "GREEN";
        } else if (color.equals("GREEN")) {
            color = "YELLOW";
        } else {
            color = "RED";
        }
    }

    public String getColor() {
        return color;
    }

    public static void main(String[] args) {

        TrafficLight t = new TrafficLight("TL-9");

        System.out.println("Color: " + t.getColor());

        t.next();
        System.out.println("Color: " + t.getColor());

        t.next();
        System.out.println("Color: " + t.getColor());

        t.next();
        System.out.println("Color: " + t.getColor());
    }
}